src/engine/Drawing/Sprite.ts

```diff
@@ -51,7 +51,7 @@ export class SpriteImpl implements Drawable {
   private _spriteCtx: CanvasRenderingContext2D = null;
   private _pixelData: ImageData = null;
   private _pixelsLoaded: boolean = false;
-  private _dirtyEffect: boolean = false;
+  private _dirtyEffect: boolean = true;
```

Comment:
I'll look at this a little closer, but that is what it means. Perhaps there is a better way to handle the new load semantics and initializing the sprite for first use

Reply:
I think this change is desirable, in the past the Excalibur promise implementation was synchronous and would immediately flip this to true at construction time. now that there is a microtask from the browser promise this state should start at true otherwise the initial pixel load will never happen.

https://github.com/excaliburjs/Excalibur/pull/1707/files/e161b5d92043a575dd3506213fe8089ab6919545#diff-40fef6d9412ae9640b0d53676193a416542b9b1c6df55ce3ccf7d0ba64b5f4a1L92